### PR TITLE
BPA Master Data configuration

### PR DESCRIPTION
**Pull request by phanikiran-wtt-egov**
Master Data configuration for 
ApplicationType
ServiceType
DocType
CheckList

Kindly review and share the feedback if any.

Thanks,
Phani

### CODE REVIEW

File: data/pb/BPA/CheckList.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "tenantId": "pb",
+    "moduleName": "BPA",
+    "CheckList": [
```
Comment (GhanshyamRawat) [inline data/pb/BPA/CheckList.json]: Change the master name, It is not a checklist, this is for docType mapping master

File: data/pb/BPA/DocType.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        {
+        "code": "OWNERIDENTITYPROOF.AADHAAR",
+        "active": true
+        },
+        {
+        "code": "OWNERIDENTITYPROOF.VOTERID",
+        "active": true
+        },
+        {
+        "code": "OWNERIDENTITYPROOF.DRIVING",
```
Comment (GhanshyamRawat) [inline data/pb/BPA/DocType.json]: Change it to driving license

File: data/pb/BPA/DocType.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        "active": true
+        },
+        {
+        "code": "OWNERADDRESSPROOF.ELECTRICITYBILL",
+        "active": true
+        },
+        {
+        "code": "OWNERADDRESSPROOF.DL",
```
Comment (GhanshyamRawat) [inline data/pb/BPA/DocType.json]: This is duplicate, please remove this.

File: data/pb/BPA/ApplicationType.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "tenantId": "pb",
+  "moduleName": "BPA",
+  "ApplicationType": [
+     {
+      "code": "BUILDING_PLAN_SCRUTINY",
+      "name":"Buildind Plan Scrutiny",
```
Comment (GhanshyamRawat) [inline data/pb/BPA/ApplicationType.json]: Remove name from master.

File: data/pb/BPA/DocType.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (GhanshyamRawat) [inline data/pb/BPA/DocType.json]: Checkin this master under common not in BPA

File: data/pb/BPA/ServiceType.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+    "tenantId": "pb",
+    "moduleName": "BPA",
+    "ServiceType": [
+       {
+        "code": "NEW_CONSTRUCTION",
+        "name":"New Construction",
```
Comment (GhanshyamRawat) [inline data/pb/BPA/ServiceType.json]: Remove name.

File: data/pb/BPA/CheckList.json (hunk 1 above)
Comment (phanikiran-wtt-egov) [reply data/pb/BPA/CheckList.json]: renaming checkList to DocTypeMapping is that ok ?

**Comment by phanikiran-wtt-egov** [comment]
Review comments fixed in the commit  https://github.com/egovernments/egov-mdms-data/pull/447/commits/7c1e95183d8926cc7e7014c7b622e2995831c3af

File: data/pb/BPA/DocTypeMapping.json [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    "tenantId": "pb",
+    "moduleName": "BPA",
+    "DocTypeMapping": [
+        {
+            "applicationType": "BUILDING_PLAN_SCRUTINY",
+            "ServiceType": "NEW_CONSTRUCTION",
+            "RiskType": "High",
+            "docTypes": [
+                {
+                    "code": "VOTERID",
```
Comment (GhanshyamRawat) [inline data/pb/BPA/DocTypeMapping.json]: It should be a namespace master as I have shared with you.

File: data/pb/BPA/DocTypeMapping.json [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+                    "required": true,
+                    "sequence":0
```
Comment (GhanshyamRawat) [inline data/pb/BPA/DocTypeMapping.json]: Why do we need sequence in master data? Can we use the order of the array it self.

File: master-config.json [hunk 9]
```diff
@@ -145,7 +145,7 @@
       ]
     },
     "DocumentType": {
-      "masterName": "StructureType",
+      "masterName": "DocumentType",
```
Comment (GhanshyamRawat) [inline master-config.json]: The structure type is another master, why you are changing this?

**Pull request merged**